### ISSUE TITLE
Cannot install "Social" app 

### ISSUE THREAD

**Issue by pod-o-mart** [nextcloud/social#1342]
Hi,

**Describe the bug**
The installation of the downloaded app "Social" https://apps.nextcloud.com/apps/social fails.

**To Reproduce**
See Nextcloud log below

**Expected behavior**
How can I make the app working?

<details>
<summary>Client details</summary>

 - OS: Ubuntu Linux
 - Browser: Firefox
 - Version: 96
 - Device: desktop
</details>

<details>
<summary>Server details</summary>

**Operating system**:
Ubuntu 18.04.6 LTS (Plesk Obsidian 18.0.40 Update 3)

**Web server:**
- memory_limit: 512M
- APCu Support:Enabled Version: 5.1.21 

**Database:**
MariaDB 10.1.48

**PHP version:**
PHP 8.0.14

**Nextcloud version:** (see Nextcloud admin page)
- Nextcloud ver.: 23.0.0.10
- Social app ver.:  0.4.2
</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
{"reqId":"YefHwuTWrMg7tQOAL-n@BAAAAAQ","level":3,"time":"2022-01-19T08:11:47+00:00","remoteAddr":"95.116.240.30","user":"martin-admin","app":"index","method":"POST","url":"/index.php/settings/apps/enable","message":"Undefined constant Doctrine\\DBAL\\Types\\Type::TEXT in file '/var/www/vhosts/podolak.net/cloud.podolak.net/apps/social/lib/Migration/Version0003Date20200611000001.php' line 228","userAgent":"Mozilla/5.0 (Windows NT 10.0; rv:91.0) Gecko/20100101 Firefox/91.0","version":"23.0.0.10","exception":{"Exception":"Exception","Message":"Undefined constant Doctrine\\DBAL\\Types\\Type::TEXT in file '/var/www/vhosts/podolak.net/cloud.podolak.net/apps/social/lib/Migration/Version0003Date20200611000001.php' line 228","Code":0,"Trace":[{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/AppFramework/App.php","line":157,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/Route/Router.php","line":302,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/base.php","line":1006,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/AppFramework/Http/Dispatcher.php","Line":158,"Previous":{"Exception":"Error","Message":"Undefined constant Doctrine\\DBAL\\Types\\Type::TEXT","Code":0,"Trace":[{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/apps/social/lib/Migration/Version0003Date20200611000001.php","line":76,"function":"createActors","class":"OCA\\Social\\Migration\\Version0003Date20200611000001","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/DB/MigrationService.php","line":454,"function":"changeSchema","class":"OCA\\Social\\Migration\\Version0003Date20200611000001","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/DB/MigrationService.php","line":417,"function":"migrateSchemaOnly","class":"OC\\DB\\MigrationService","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/Installer.php","line":154,"function":"migrate","class":"OC\\DB\\MigrationService","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/apps/settings/lib/Controller/AppSettingsController.php","line":452,"function":"installApp","class":"OC\\Installer","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/AppFramework/Http/Dispatcher.php","line":217,"function":"enableApps","class":"OCA\\Settings\\Controller\\AppSettingsController","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/AppFramework/Http/Dispatcher.php","line":126,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/AppFramework/App.php","line":157,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/private/Route/Router.php","line":302,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/lib/base.php","line":1006,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/vhosts/podolak.net/cloud.podolak.net/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/vhosts/podolak.net/cloud.podolak.net/apps/social/lib/Migration/Version0003Date20200611000001.php","Line":228},"CustomMessage":"--"}}
```
</details>

Cheers,

Martin

**Comment by violoncelloCH** [comment]
should be fixed by #1369

**Comment by Quix0r** [comment]
> I have a similar problem:
> 
> ```
> [index] Erreur: Exception: Undefined class constant 'TEXT' in file '/var/www/html/nc/apps/social/lib/Migration/Version0003Date20200611000001.php' line 228 at <<closure>>
> 
> 0. /var/www/html/nc/lib/private/AppFramework/App.php line 172
>    OC\AppFramework\Http\Dispatcher->dispatch()
> 1. /var/www/html/nc/lib/private/Route/Router.php line 298
>    OC\AppFramework\App::main()
> 2. /var/www/html/nc/lib/base.php line 1047
>    OC\Route\Router->match()
> 3. /var/www/html/nc/index.php line 36
>    OC::handleRequest()
> 
> POST /nc/index.php/settings/apps/enable
> from 147.94.235.37 by laurent at 2022-11-07T07:57:57+00:00
> ```
> 
> I am running nextcloud 25.0 and willing to give more details if it useful.

Same problem here, too. The app is downloaded but not enabled. 25.0.13 here
